src/test/java/com/adyen/MarketPayNotificationTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2017 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+package com.adyen;
+
+import com.adyen.model.marketpay.notification.*;
+import com.adyen.service.Notification;
+import org.junit.Test;
+
+public class MarketPayNotificationTest extends BaseTest {
+
+    @Test
+    public void TestCreateNotificationConfiguration() throws Exception {
+        Client client = createMockClientFromFile("mocks/marketpay/notification/createNotificationConfiguration.json");
```

Comment:
Should we use "create-notification-configuration-success.json" for the json files like in the other mocks?